src/status_im2/contexts/syncing/enter_sync_code/view.cljs

```diff
@@ -0,0 +1,84 @@
+    (fn []
+      (clipboard/get-string #(reset! clipboard %))
+      (let [invalid?           false
+            show-paste-button? (string/blank? @default-value)]
+        [:<>
+         [rn/view
+          {:style style/label-container}
+          [quo/text
+           {:style  style/label-pairing
+            :weight :medium
+            :size   :paragraph-2}
+           (i18n/label :t/type-pairing-code)]]
+         [rn/view {:style style/container-text-input}
+          [rn/view {:style (style/text-input-container invalid?)}
+           [rn/text-input
+            {:style                  (style/text-input)
+             :default-value          @default-value
+             :placeholder            (i18n/label :t/type-some-chat-key)
+             :on-change-text         (fn [scan-code]
+                                       (reset! default-value scan-code)
+                                       (reset! submit-button-disabled? false))
+             :blur-on-submit         true
+             :return-key-type        :done
+             :accessibility-label    :enter-sync-code-input
+             :auto-capitalize        :none
+             :placeholder-text-color colors/white-opa-40
+             :multiline              true}]
+           (if show-paste-button?
+             [rn/view {:style style/button-paste}
+              [quo/button
+               {:on-press (fn [_]
+                            (reset! default-value @clipboard)
+                            (reset! submit-button-disabled? false))
+                :type     :outline
+                :size     24}
+               (i18n/label :t/paste)]]
+
+             [rn/touchable-opacity
+              {:accessibility-label :input-right-icon
+               :style               style/right-icon-touchable-area
+               :on-press            (fn [_]
+                                      (reset! default-value nil)
+                                      (reset! submit-button-disabled? true))}
+              [icon/icon :i/clear style/clear-icon]])]]
+         [rn/view {:style style/continue-button-container}
+          [quo/button
+           {:type                :primary
+            :disabled            @submit-button-disabled?
+            :theme               :dark
+            :customization-color profile-color
+            :style               style/button-confirm
+            :on-press            (fn [_]
+                                   (if (sync-utils/valid-connection-string? @default-value)
+                                     (debounce/debounce-and-dispatch
+                                      [:syncing/input-connection-string-for-bootstrapping
+                                       @default-value]
+                                      300)
+                                     (rf/dispatch [:toasts/upsert
+                                                   {:icon :i/info
+                                                    :icon-color colors/danger-50
```

Comment:
not related to your pr but toasts is setup wrong here - colors should not be set in use, it should be encapsulated by the component and set using configuration.

<img width="242" alt="Screenshot 2023-08-03 at 08 33 05" src="https://github.com/status-im/status-mobile/assets/22799766/f3780ab7-8cc5-41ea-b68d-e120af509f14">


:icon-color colors/danger-50

Reply:
Yes, Jamie, I noticed it while adding the theme context. 
We need to ask the design team about the types of colours we will display (or should be supported) in the toast (like `:success`, `:error`, `:warning`, and `:info`), and then we can remove the `icon-color` prop.

Reply:
Oh those colors are already there in the design system. I believe component just needs to be updated so it can be used properly 👍

Reply:
Thanks for cc'ing me @J-Son89. Good point!

Reply:
Do we already have an issue to track this? If not I can create one

Reply:
Thanks @siddarthkay! As far as i know there is no issue created for this. Might be worth a quick check but would be great if you can create it 🙌🏼

Reply:
https://github.com/status-im/status-mobile/issues/16878